low-level/solution/lr2/support_bundle/sspl_support_bundle

```diff
@@ -195,7 +195,7 @@ class SupportBundle(object):
 
         if file_path:
             if os.path.exists(file_path):
-                self.tmp_bundle_path = os.path.normpath(file_path)
+                self.tmp_bundle_path = os.path.normpath(file_path+"sspl")
```

Comment:
Unless we use default path, **file_path** will be coming from user who may not add '/' at the end of the path. So, if I give path `/var/cortx` for sspl support bundle the resultant path will become `/var/cortxsspl`
Add `/sspl`, `normpath` will take care of any extra '/' in the path.

Reply:
I don't feel it is required.
I have tested it in VM as well as the hw setup and it is working as expected.

Reply:
Can you please try below command and see if you are getting correct results:
`/opt/seagate/cortx/sspl/bin/sspl_support_bundle create test_bundle /tmp/sat001 -console`

Reply:
Yes @jjxsg, for the above case it is appending an SSPL at the end. made required chnages.
Done.